docs/docs/sourcing-from-kentico-kontent.md

```diff
@@ -6,17 +6,17 @@ In this guide, you'll go through how to quickly source content for your Gatsby s
 
-You can also take advantage of Kentico Kontent's ability to deliver content in multiple languages and create relationships among various content with linked items. However you structure your content in Kentico Kontent, the [source plugin](https://github.com/Kentico/gatsby-source-kontent) will ensure the proper nodes are created for your Gatsby site.
+You can also take advantage of Kentico Kontent's ability to deliver content in multiple languages and create relationships among various content with linked items. However you structure your content in Kentico Kontent, the [source plugin](https://github.com/Kentico/kontent-gatsby-packages/tree/master/packages/gatsby-source-kontent) will ensure the proper nodes are created for your Gatsby site.
 
-_Note: For this guide, you'll work from scratch from the [Gatsby default starter](https://github.com/gatsbyjs/gatsby-starter-default) to get a feel for how sourcing from Kentico Kontent works. If you'd like to look at an example of a complete basic site, have a look at the [Kentico Cloud Gatsby starter site](https://github.com/Kentico/gatsby-starter-kontent), which includes worked examples for querying content._
+_Note: For this guide, you'll work from scratch from the [Gatsby default starter](https://github.com/gatsbyjs/gatsby-starter-default) to get a feel for how sourcing from Kentico Kontent works. If you'd like to look at an example of a complete basic site, have a look at the [Kentico Kontent Gatsby starter site](https://github.com/Kentico/gatsby-starter-kontent), which includes worked examples for querying content. Or you could also check out the [Kontent starter based on Lumen template](https://github.com/Kentico/gatsby-starter-kontent-lumen), which is a minimal, lightweight and mobile-first starter for creating blogs using Gatsby and Kentico Kontent._
 
 ## Setup
 
 ### Kentico Kontent
 
-The first thing to do, if you haven't already done so, is [sign up for a Kentico Kontent account](https://app.kontent.ai/sign-up). This will automatically start a free 30-day trial with all Kentico Kontent features. At any point during the trial or after, you can switch to a Starter plan (which always starts as free) or a higher plan with more features.
+The first thing to do, if you haven't already done so, is [sign up for a Kentico Kontent account](https://app.kontent.ai/sign-uputm_medium=gatsbyjsorg&utm_source=guide). This will automatically start a free 30-day trial with all Kentico Kontent features. At any point during the trial or after, you can switch to a Starter plan (which always starts as free) or a higher plan with more features.
```

Comment:
```suggestion
The first thing to do, if you haven't already done so, is [sign up for a Kentico Kontent account](https://app.kontent.ai/sign-up?utm_medium=gatsbyjsorg&utm_source=guide). This will automatically start a free 30-day trial with all Kentico Kontent features. At any point during the trial or after, you can switch to a Starter plan (which always starts as free) or a higher plan with more features.
```

Looks like this is missing a "?" 🤔

Reply:
d950fcf